### PR TITLE
Feature/input component

### PR DESCRIPTION
**Pull request by Sujal0007**
Added a Input Field React component .

### CODE REVIEW

File: src/Minicomponents/InputField.jsx [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  ...props
+}) {
+  return (
+    <div className={className}>
+      {label && <label>{label}</label>}
+      <input
+        id={id || name}
+        type={type}
+        value={value}
+        onChange={onChange}
```
Comment (Wannacryy24) [inline src/Minicomponents/InputField.jsx]: function missing

File: src/Minicomponents/InputField.jsx [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+        placeholder={placeholder}
+        name={name}
+        required={required}
+        className={className}
+        {...props}
```
Comment (Wannacryy24) [inline src/Minicomponents/InputField.jsx:124]: height and width can also be changed , so we can add that too

Comment (Sujal0007) [reply src/Minicomponents/InputField.jsx:124]: Sure! I'll update the Input component to include customizable height and width as props.

Comment (Sujal0007) [reply src/Minicomponents/InputField.jsx:124]: I've updated the Input component to include customizable height and width as props. Let me know if there's anything else you'd like to add or modify!

Comment (Wannacryy24) [reply src/Minicomponents/InputField.jsx:124]: wait

**Pull request merged**